packages/desktop/scripts/copy_wheels.ts

```diff
@@ -20,15 +26,18 @@ async function main() {
     stliteKernelPyDir,
     "stlite-lib/dist/stlite_lib-0.1.0-py3-none-any.whl",
   );
+  const streamlitVersion = process.env.STREAMLIT_VERSION || "1.41.0";
```

Comment:
What provides the `STREAMLIT_VERSION` environment variable?
Looks like it's now working just because of the fallback value. Delete the fallback value for the "fail early" principle.